docs/widget/migration-guide.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: 'Widget V2 Migration Guide'
+description: 'This guide provides a concise overview of the changes needed to migrate your existing Widget V1 implementation to Widget V2.'
+---
+
+## 1. Update Dependency
+    <CodeGroup>
+    ```shell NPM
+    npm install @skip-go/widget@latest
+    ```
+    ```shell Yarn
+    yarn add @skip-go/widget@latest
+    ```
+    </CodeGroup>
+
+## 2. `theme` Prop Changes
+
+### More Customization Options
+ You can pass either `light`, `dark`, or a custom theme object with granular 
+control over the widget's appearance.
+
+**Before:**
+```tsx
+  <Widget
+    theme='{
+        "backgroundColor": "#191A1C",
+        "textColor": "#E6EAE9",
+        "borderColor": "#363B3F",
+        "brandColor": "#FF4FFF",
+        "highlightColor": "#1F2022"
+    }'
+  />
+```
+**After:**
```

Comment:
btw, there's a new "brandColor" prop on the widget I think we should consider removing brandColor from the theme prop examples (even though it's valid) so that it doesn't confuse people and push people towards adding brandColor outside of the theme prop

the reason I had to add this was because if we want you to be able to use theme="light" / theme="dark" prop while also passing a brandColor

Reply:
ok added in an line comment, good shout!